### PR TITLE
Adding the phabricator checker!

### PR DESCRIPTION
**Pull request by jshirley**
This adds a custom phabricator checker for us to use, tested on qa-phab.

r? @dylanvee (since you know the python) || @gphat (since you know obs)

### CODE REVIEW

**Comment by dylanvee** [comment]
LGTM, modulo nits and things in @gphat's area of expertise!

File: checks.d/phabricator.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Fetch metrics from Phabricator, using our MetricsExtension (not yet available)
+
+# stdlib
+from collections import defaultdict, namedtuple
```
Comment (gphat) [inline checks.d/phabricator.py]: `defaultdict` isn't used

File: checks.d/phabricator.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+import time
```
Comment (gphat) [inline checks.d/phabricator.py]: `time` unused

File: checks.d/phabricator.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+import urlparse
+
+# 3p
+import requests
+
+# project
+from checks import AgentCheck
+from config import _is_affirmative
```
Comment (gphat) [inline checks.d/phabricator.py]: this doesn't look used?

**Comment by gphat** [comment]
:+1: other than nits about unnecessary includes, and assuming it works in your testing on a box!

**Pull request merged**